custom_components/elkbledom/elkbledom.py

```diff
@@ -165,7 +168,7 @@ def effect(self):
 
     @retry_bluetooth_connection_error
     async def set_white(self, intensity: int):
-        await self._write([0x7e, 0x00, 0x01, intensity, 0x00, 0x00, 0x00, 0x00, 0xef])
+        await self._write([0x7e, 0x00, 0x01, int(intensity*100/255), 0x00, 0x00, 0x00, 0x00, 0xef])
```

Comment:
Unsure if this is true for other devices... my one seems to have a range of 0 - 0x64; i.e. 0x64 is max brightness, 0x65 is same as 0x0.